modules/next_stage/README.md

```diff
@@ -0,0 +1,51 @@
+# Next Stage
+
+## Purpose
+
+The `next_stage` module provides a method of start the "next stage"
```

Comment:
suggest "... method of starting the ..." or "... method to start the ... "

Reply:
I went with "to" since it requires less formatting change..